src/app/global-vars.service.ts

```diff
@@ -1397,24 +1230,23 @@ export class GlobalVarsService {
       if (environment.production) {
         this.localNode = hostname;
       } else {
-        this.localNode = `${hostname}:18001`;
+        this.localNode = `https://node.deso.org`;
```

Comment:
to check

Reply:
Whoops yeah this change should be reverted (though it only affects local dev)